src/extensions/default/bramble/lib/UI.js

```diff
@@ -90,6 +90,14 @@ define(function (require, exports, module) {
             PreferencesManager.set("wordWrap", wordWrap);
         }
 
+        var autoCloseTags = BrambleStartupState.ui("autoCloseTags");
+        if(typeof autoCloseTags === "object") {
+            if(autoCloseTags === null) {
+                autoCloseTags = { whenOpening: true, whenClosing: true, indentTags: [] };
```

Comment:
We can reduce this code a lot.  A more typical way to do this in JS is to do:

```js
var autoCloseTags = BrambleStartupState.ui("autoCloseTags") || { whenOpening: true, whenClosing: true, indentTags: [] };
```

And you can drop both of the `if` checkso n 94, 95.